OneCommunityGlobal/HGNRest
# Carlos new pie chart hour visualization by team member  project reports

This Backend PR is related to supporting proposal changes in Frontend [PR2409](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2409)

Description
The graph currently displays weekly committed hours by project member. However, it does not show the actual hours worked by member on each project.

(PRIORITY HIGH) Jae: Add “Hours by Team Member” Visualization to the Projects Reports page (WIP Carlos)

Main changes explained:
in getTimeEntriesForSpecifiedProjectadd, add firstName lastName and isActive properties in 'TimeEntry' Schema from 'userProfile' Schema with will be received in the fronted.
How to test:
check into current branch
do npm install and ... to run this PR locally
Clear site data/cache
log as Owner, manager or admin user
Folllow Frontend intructions
Requested job:
task

video before the change:

https://www.loom.com/share/7be14c4aec464c5fbb9669a11c4d63b5?sid=586d7ba2-8f94-4a4a-99e6-6725b31d4236

video after the change:

https://www.loom.com/share/889ae08438b94daca7d4f63f9905cb07?sid=2039627e-1faa-43ae-84a6-6d5f3b8a53af

Note:
You must add a project for at least one user ( set as a poject member)
You must log hours for this member(s) in this project.
Go to dashboard => report => type the project => PieChart => pick the charts
4.See videos above for more info
